Cargo.toml

```diff
@@ -94,6 +93,75 @@ exclude = [
   'docs/rust_wasi_markdown_parser'
 ]
 
+[workspace.package]
+version = "2.0.0"
+authors = ["The Wasmtime Project Developers"]
+edition = "2021"
+
+[workspace.dependencies]
+wasmtime = { path = "crates/wasmtime", version = "2.0.0", default-features = false }
+wasmtime-cache = { path = "crates/cache", version = "=2.0.0" }
+wasmtime-cli-flags = { path = "crates/cli-flags", version = "=2.0.0" }
+wasmtime-cranelift = { path = "crates/cranelift", version = "=2.0.0" }
+wasmtime-environ = { path = "crates/environ", version = "=2.0.0" }
+wasmtime-fiber = { path = "crates/fiber", version = "=2.0.0" }
+wasmtime-types = { path = "crates/types", version = "=2.0.0" }
+wasmtime-jit = { path = "crates/jit", version = "=2.0.0" }
+wasmtime-jit-debug = { path = "crates/jit-debug", version = "=2.0.0" }
+wasmtime-runtime = { path = "crates/runtime", version = "=2.0.0" }
+wasmtime-wast = { path = "crates/wast", version = "=2.0.0" }
+wasmtime-wasi = { path = "crates/wasi", version = "2.0.0" }
+wasmtime-wasi-crypto = { path = "crates/wasi-crypto", version = "2.0.0" }
+wasmtime-wasi-nn = { path = "crates/wasi-nn", version = "2.0.0" }
+wasmtime-component-util = { path = "crates/component-util", version = "=2.0.0" }
+wasmtime-component-macro = { path = "crates/component-macro", version = "=2.0.0" }
+wasmtime-asm-macros = { path = "crates/asm-macros", version = "=2.0.0" }
+component-test-util = { path = "crates/misc/component-test-util" }
+component-fuzz-util = { path = "crates/misc/component-fuzz-util" }
+wiggle = { path = "crates/wiggle", version = "=2.0.0" }
+wiggle-macro = { path = "crates/wiggle/macro", version = "=2.0.0" }
+wiggle-generate = { path = "crates/wiggle/generate", version = "=2.0.0" }
+wasi-common = { path = "crates/wasi-common", version = "2.0.0" }
+wasi-tokio = { path = "crates/wasi-common/tokio", version = "2.0.0" }
+wasi-cap-std-sync = { path = "crates/wasi-common/cap-std-sync", version = "=2.0.0" }
+wasmtime-fuzzing = { path = "crates/fuzzing" }
+
+cranelift-wasm = { path = "cranelift/wasm", version = "0.89.0" }
```

Comment:
This doesn't break using cranelift as path dependency from a different workspace, right?

Reply:
I don't believe so, no